playbooks/roles/ocp-customization/defaults/main.yaml

```diff
@@ -4,4 +4,5 @@
 workdir: ~/ocp4-workdir
 rhcos_kernel_options: []
 sysctl_tuned_options: false
+powervm_rmc: true
```

Comment:
the powervm_rmc should be set to true only for PowerVM, for KVM is should be false.

Reply:
@cs-zhang, do you know a way to detect powerVM or KVM ? What do you suggest ?

Reply:
We should be able to change the `powervm_rmc` to `false` when using this in the KVM playbook. Also I think this will help in KVM case for DLPAR operations.

Reply:
We don't need it for KVM DLPAR as we don't use HMC at all.... 
We can set it to `false` for a KVM installation, and even if it is set to `true`, it should not be a problem to have the rmc container started (except extra resources consumption)